src/Math-Tests-AutomaticDifferenciation/PMDualNumberTest.class.st

```diff
@@ -19,18 +19,16 @@ PMDualNumberTest >> assertEquality: aDualNumber and: anotherDualNumber [
 
 { #category : #running }
 PMDualNumberTest >> setUp [
-zero:=PMDualNumber value: 0 eps: 1.
-zeroc:=PMDualNumber value: 0 eps: 0.
-one:=PMDualNumber value: 1 eps: 1.
-onec:=PMDualNumber value: 1.
-three :=PMDualNumber value: 3.0 eps: 1.
-
+	zero := PMDualNumber value: 0 eps: 1.
+	zeroc := PMDualNumber value: 0 eps: 0.
```

Comment:
What is `zeroc`

Reply:
I guess the c means constant, but I don't like the name constant.